src/test/resources/com/puppycrawl/tools/checkstyle/filters/strategy/RegexpOnFilename/newline/expected.txt

```diff
@@ -1,4 +1,3 @@
-Starting audit...
-[ERROR] /Users/hgh/Downloads/my_file/IDEA_workspace/patchfiltergeneratepatch/src/main/java/Checker/RegexpOnFilename/Test3.java:1: File match folder pattern '' and file pattern '\.java$'. [RegexpOnFilename]
-Audit done.
-
+Test3.java:1: File match folder pattern '' and file pattern '\.java$'.
+Test2.java:1: File match folder pattern '' and file pattern '\.java$'.
+Test.java:1: File match folder pattern '' and file pattern '\.java$'.
```

Comment:
Expected violations before and after do not match. Probably that's why your test fail.

Reply:
@baratali ...but It is ok on my local.

Reply:
done